src/container-manager.ts

```diff
@@ -0,0 +1,50 @@
+import { spawn, spawnSync } from 'child_process'
+import { writeFileSync } from 'fs';
+import { mkdtemp, rm } from 'fs/promises';
```

Comment:
Oh, I didn't know about these new additions to Node! We could use `rm` in `deleteJamsocketConfig` instead of `unlink`, too.

Reply:
Also, just noticing that `rm` is only available in Node 14.14 and later which I think came out at the end of 2020. So there might be a few folks who have not upgraded yet who end up using this. Not saying we need to change anything here, but good to keep an eye on what minimum version of Node we expect users to be on

Reply:
I tried replacing with `rmdir` using its `recursive` flag (which existed since v12.10.0), but node prints a warning that it will be removed in a future version. Given the choice between backwards-compatible and forwards-compatible, I say we go with forwards-compatible.

Reply:
I think `unlink` might be the most backwards- and forwards-compatible way. But I honestly don't care too much about backwards-compatibility. I think it's fine to make 14.14+ the expected Node version